### PR TITLE
implement linked list class

### PR DESCRIPTION
**Pull request by justin--**
I implemented a linked list 

steps in git : 
git checkout master
git pull
git checkout -b {YOUR NEW BRANCH}
... DO SOME WORK
.. COMMIT WORK
git push

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -2,46 +2,14 @@
 <project version="4">
   <component name="ChangeListManager">
     <list default="true" id="1de81bfc-b592-47ef-8e4e-30915d02c3e2" name="Default Changelist" comment="">
```
Comment (justin--) [inline .idea/workspace.xml:4]: I didnt mean to put this stuff here but i left it anyways

File: data-structures/LinkedList.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
+class Node:
+    def __init__(self, value, next_node=None):
+        self.value = value
+        self.next = next_node
+
+
+class LinkedList:
```
Comment (justin--) [inline data-structures/LinkedList.py:7]: This class can be used as a list

File: data-structures/LinkedList.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        self.head = None
+        self.total = 0
+
+    def add(self, value):
+        """ Add to the end of the list """
+        new_element = Node(value)
+        if not self.head:
+            self.head = new_element
+        else:
+            current = self.head
```
Comment (dhita-irma) [inline data-structures/LinkedList.py:18]: Can we change 'current' to 'pointer'?

Comment (justin--) [reply data-structures/LinkedList.py:18]: We can but i feel like current makes sense as a way of tracking. If it bugs you i can change to pointer.

Comment (dhita-irma) [reply data-structures/LinkedList.py:18]: Okay, I agree

**Review by dhita-irma — APPROVED** [review]
LGTM!

**Pull request merged**